extensions/warp-ipfs/src/store/files.rs

```diff
@@ -34,7 +34,8 @@ use crate::{
 };
 
 use super::{
-    document::root::RootDocumentMap, event_subscription::EventSubscription,
+    document::root::{self, RootDocumentMap},
```

Comment:
Causing linting to fail

```suggestion
    document::root::RootDocumentMap,
```